squad/core/models.py

```diff
@@ -691,14 +741,42 @@ def __str__(self):
         return self.job_id and ('#%s' % self.job_id) or ('(%s)' % self.id)
 
 
+@receiver(pre_delete, sender=TestRun)
```

Comment:
@chaws this is to delete the files once copying concludes I assume? I will read with fresher eyes tomorrow morning.

Reply:
That gets executed when a `TestRun` object is being deleted, i.e. somewhere in `testrun.delete()`